README.md

```diff
@@ -384,10 +384,87 @@ if (result.isIntervalResult()) {
     }
 }
 ```
+To perform a [Multi-Analysis](https://keen.io/docs/api/#multi-analysis), use the MultiAnalysis.Builder instead. An instance of MultiAnalysisResult will be returned unless Group By and/or Interval parameters are included, in which case the MultiAnalysisResult(s) will be nested inside an IntervalResult and/or a GroupByResult, just like any other QueryResult for other single analysis types when grouping/intervals are applied:
+
+``` java
+final MultiAnalysis multiAnalysis = new MultiAnalysis.Builder()
+        .withCollectionName("the_collection")
```

Comment:
I think `withCollectionName` is fine if we are doing a breaking version bump, as long as we use that everywhere.